pandas_hw.py

```diff
@@ -0,0 +1,79 @@
+# First things first, we need to import all necessary modules which are listed in the requirements.txt. This is crucial!
+# 1. We have 2 functions to read our data
+## 1.1. This function takes a bed file, gives it appropriate headers and returns it in a nicely workable view.
+def read_bed6(input_file):
+    resbed = pd.read_csv(input_file, sep='\t', header=None)
+    resbedheader = ['chrom', 'chromStart', 'chromEnd', 'name', 'score', 'strand']
+    resbed.columns = resbedheader[:len(resbed.columns)]
+    return resbed
+## Example usage with our bed file  
+bedfile = read_bed6('alignment.bed')
+
+## 1.2. This function takes a gff file, gives it appropriate headers and returns it in a nicely workable view.
+def read_gff(input_file):
+    resgff = pd.read_csv(input_file, header=None, skiprows=1, sep = '\t')
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    resgff = pd.read_csv(input_file, header=None, comment="#", sep='\t')
```